.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug Report
+about: Report a bug encountered while using Kubernetes release engineering tooling
+labels: kind/bug, sig/release, area/release-eng
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+
+If the matter is security related, please disclose it privately via https://kubernetes.io/security/
+-->
+
+#### What happened:
+
+#### What you expected to happen:
+
+#### How to reproduce it (as minimally and precisely as possible):
+
+#### Anything else we need to know?:
+
+#### Environment:
+
+- Cloud provider or hardware configuration:
```

Comment:
change cloud provider to "container runtime" ?

Reply:
:+1: